### DISCUSSION TITLE
rettwet  5000 points problem

### DISCUSSION THREAD

**Discussion by hizen1984** [IBAX-io/go-ibax#1697] [Bug Submission]
hello, the last rettweet task with 5000 points is not added to the ibax points even though it is done.   PR (your servers may be overloaded, turn it off and on it always works).

**Comment by muco333** [comment]
https://twitter.com/KavusMucahit/status/1488035141880619009 

give support for this issue.

**Comment by muco3327** [comment]
https://twitter.com/KavusMucahit/status/1488035141880619009

give support for this issue.
